FirebaseSessions/Sources/SessionStartEvent.swift

```diff
@@ -147,4 +151,65 @@ class SessionStartEvent: NSObject, GDTCOREventDataObject {
       return firebase_appquality_sessions_LogEnvironment_LOG_ENVIRONMENT_AUTOPUSH
     }
   }
+
+  private func convertNetworkType(networkType: GULNetworkType)
+    -> firebase_appquality_sessions_NetworkConnectionInfo_NetworkType {
+    switch networkType {
+    case .WIFI:
+      return firebase_appquality_sessions_NetworkConnectionInfo_NetworkType_WIFI
+    case .mobile:
+      return firebase_appquality_sessions_NetworkConnectionInfo_NetworkType_MOBILE
+    case .none:
+      return firebase_appquality_sessions_NetworkConnectionInfo_NetworkType_DUMMY
+    @unknown default:
+      return firebase_appquality_sessions_NetworkConnectionInfo_NetworkType_DUMMY
+    }
+  }
+
+  private func convertMobileSubtype(mobileSubtype: String)
+    -> firebase_appquality_sessions_NetworkConnectionInfo_MobileSubtype {
+    var subtype: firebase_appquality_sessions_NetworkConnectionInfo_MobileSubtype
+
+    #if TARGET_OS_IOS
```

Comment:
Watch out - this is an Objective-C macro, so these tests aren't being run. The equivalent macro for Swift is `#if os(iOS)`

Reply:
I tried applying the the Swift Macro - But that did not really work. So, had to settle down with the macro `TARGET_OS_IOS`.

Reply:
what issue did you run into with Sam's suggestion? I don't think TARGET_OS_IOS expands correctly in a Swift context because, as Sam said, it's an Objective-C macro.

Reply:
nevermind-- saw they were changed in 505a108!